research/object_detection/model_main.py

```diff
@@ -106,4 +104,6 @@ def main(unused_argv):
 
 
 if __name__ == '__main__':
+  flags.mark_flag_as_required('model_dir')
+  flags.mark_flag_as_required('pipeline_config_path')
```

Comment:
Maybe move these before `FLAGS = flags.FLAGS` (line 55). I think this is more consistent with other examples.